envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
+  hwraid:
+    enabled: False
+
+logging:
+  enabled: true
+  follow:
+    enabled: true
+    global_fields:
+      customer_id: "unknown"
+      cluster_name: "unknown"
+    logs:
+      - paths:
+          - /var/log/messages
+        fields:
+          type: syslog
+  forward:
+    host: 127.0.0.1
+    port: 4560
+
+haproxy:
+  stats_group: root
+
 keystone:
-  ldap_domain:
-    enabled: True
-    domain: users
+  uwsgi:
+    method: port
+
+nova:
+  libvirt_type: kvm
 
 serverspec:
   enabled: True
 
+heat:
+  enabled: True
+  plugin_dirs:
```

Comment:
Why are we adding this here?  these heat overrides are not applicable to ubuntu. Ubuntu has different path.